<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IFOCUSODC-PC47" name="API_VootKids.UpdateProfile" tests="1" failures="1" timestamp="30 Apr 2018 12:03:23 GMT" time="19.608" errors="0">
  <testcase name="SignUp_Kids" time="19.608" classname="API_VootKids.UpdateProfile">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Unable to retrieve updated profile.] but found [This Account ID is invalid, please use valid account ID.],
	expected [Unable to retrieve updated profile.] but found [This Account ID is invalid, please use valid account ID.],
	expected [Unable to retrieve updated profile.] but found [This unique record Id is invalid, please enter a valid unique record ID.],
	expected [Unable to retrieve updated profile.] but found [This unique record Id is invalid, please enter a valid unique record ID.],
	expected [The profile.name may only contain letters.] but found [OK]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Unable to retrieve updated profile.] but found [This Account ID is invalid, please use valid account ID.],
	expected [Unable to retrieve updated profile.] but found [This Account ID is invalid, please use valid account ID.],
	expected [Unable to retrieve updated profile.] but found [This unique record Id is invalid, please enter a valid unique record ID.],
	expected [Unable to retrieve updated profile.] but found [This unique record Id is invalid, please enter a valid unique record ID.],
	expected [The profile.name may only contain letters.] but found [OK]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at API_VootKids.UpdateProfile.SignUp_Kids(UpdateProfile.java:252)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SignUp_Kids -->
</testsuite> <!-- API_VootKids.UpdateProfile -->
